examples/flows/standard/basic-with-embedding-tool/flow.dag.yaml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
+inputs:
+  text:
+    type: string
+    default: Python Hello World!
+outputs:
+  output:
+    type: object
+    reference: ${embed_text.output}
+nodes:
+  - name: embed_text
+    type: python
+    source:
+      type: package
+      tool: promptflow.tools.embedding.embedding
```

Comment:
This example looks too dummy, 
Could we use a more real case to demo the usage?

Reply:
Hi, could you please give an example of what you would like to see here? 

I can 
- Increase the complexity of the example strings in `data.jsonl`.
- Add additional steps to the flow, for example: embed two different strings, then perform a comparison of the two embeddings.